source/aspnetintegration.rst

```diff
@@ -263,3 +266,210 @@ A typical solution to this problem is to split up the class into multiple smalle
 Simple Injector :ref:`promotes <Push-developers-into-best-practices>` best practices, and because of downsides described above, we consider the use of the `[FromServices]` attribute *not* to be a best practice. This is why we choose not to provide out-of-the-box support for injecting Simple Injector registered dependencies into controller actions. 
 
 In case you still feel method injection is the best option for you, you can plug in a custom `IModelBinderProvider` implementation returning a custom `IModelBinder` that resolves instances from Simple Injector.
+
+.. _using-simpleinjector-with-aspnetcore-ihostedservices:
```

Comment:
Let's change the title (to something similar) to:

> Working with Hosted Services

Also change the tag (the `.. _`) accordingly.

Reasoning for this is as follows:

* Stating "Simple Injector" is redundant, as this is the Simple Injector documentation :)
* Stating that this is for ASP.NET Core is redundant, as this is a section of the ASP.NET Core integration page
* Stating that `IHostedServices` are new in ASP.NET Core 2.0 might be useful in the actual text, but not so much in the title, as there are no IHostedServices prior to 2.0 and this documentation will likely be read by people using ASp.NET Core 3, 4, 5, 6, etc.
* I chose the "Working with" primarily because most of the sections have a title starting with "working with", but it also quite nicely describes what the user intends to do.
* chose the wording "Hosted Services" over "IHostedServices" as there is no thing as `IHostedServices`, but only `IHostedService`. `IHostedService`, however, is more the technical implementation, while the "hosted service" is the actual thing.

Feel free to comment on.